### DISCUSSION TITLE
Suggestion for calendar view to go back in time

### DISCUSSION THREAD

**Discussion by cococoot** [trakt/showly#393] [Ideas / Feature Requests]
I have added a couple pics. On the calendar section, swipe down to view episodes for tv shows prior to the current day. Also, give option to mark as watched. 

![Screenshot_20210124-154841_Google Play Store.jpg](https://user-images.githubusercontent.com/61015333/105643156-3ee30680-5e5c-11eb-80ea-1bdb9fcd5ba2.jpg)

![Screenshot_20210124-154841_Google Play Store.jpg](https://user-images.githubusercontent.com/61015333/105643164-46a2ab00-5e5c-11eb-8b3d-150921797fc6.jpg)

**Comment by cococoot** [comment]
New pic. ![Screenshot_20210124-154905_Google Play Store.jpg](https://user-images.githubusercontent.com/61015333/105643193-6639d380-5e5c-11eb-80da-812f8c486b45.jpg)

**Reply by mtalexan** [reply]
This could be configurable for a maximum number of days, or not, but if it's possible to go back in time more than a couple days then a button for "jump to today" would probably be needed as well. 
Currently if I'm looking way in the future I can just quickly scroll to the top to get back to today, but with past history also available the "top" is no longer today.

I'd also like to point out that many users make use of the app to identify what aired recently so they can stream it rather than knowing what's coming up and streaming it immediately when it airs. That would recommend this feature so it's easy to see what recently aired.

**Comment by blackholeearth** [comment]
This has been implemented - can close now 

Duplicate #488

![Zombo_calendar_past_future](https://github.com/michaldrabik/showly-2.0/assets/7512105/221f0d38-01cb-4e18-91a2-93ed21572aa7)
